### ISSUE TITLE
Carthage doesn't build dynamic framework schemes (FBSDKLoginKit, FBSDKCoreKit, etc)

### ISSUE THREAD

**Issue by ValCanBuild** [facebook/facebook-swift-sdk#434]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-swift-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `10.2.1`
- Swift Version: `5.0`
- Installation Platform & Verison: `Carthage` version `0.33.0`
## Expected Results

Have the dynamic frameworks be fetched and built with Carthage.

## Actual Results

When doing a `carthage --update` the dynamic framework schemes do not get built and the frameworks are not available, which prevents app from compiling as it can't find `FBSDKLoginKit`

## Steps to Reproduce

Pull in the latest version of swift sdk

`github "facebook/facebook-sdk-swift" ~> 0.7.0`

Example input of running  `carthage --update` when it worked:
```
*** Checking out facebook-sdk-swift at "342109d99e803b0d33020163040d6ab33c27145b"
*** Checking out facebook-objc-sdk at "7b400fab9a0774ecf0cbc504a47af944d863917a"
*** Building scheme "FBSDKCoreKit-Dynamic" in FacebookSDK.xcworkspace
*** Building scheme "FBSDKLoginKit-Dynamic" in FacebookSDK.xcworkspace
*** Building scheme "FBSDKPlacesKit-Dynamic" in FacebookSDK.xcworkspace
*** Building scheme "FBSDKShareKit-Dynamic" in FacebookSDK.xcworkspace
*** Building scheme "FacebookCore" in FacebookSwift.xcworkspace
*** Building scheme "FacebookLogin" in FacebookSwift.xcworkspace
*** Building scheme "FacebookShare" in FacebookSwift.xcworkspace
```

Example input of running it now, notice that the `-Dynamic` frameworks from the `FacebookSDK` workspace are not present:

```
*** Checking out facebook-sdk-swift at "v0.7.0"
*** Checking out facebook-objc-sdk at "v5.0.0"
*** Building scheme "FacebookCore" in FacebookSwift.xcworkspace
*** Building scheme "FacebookShare" in FacebookSwift.xcworkspace
*** Building scheme "FacebookLogin" in FacebookSwift.xcworkspace
```

**Comment by lenli** [comment]
We have this same issue.  We were able to work around this by:
1)  Removing Cartfile.resolved and clearing the Carthage `Build` and `Checkouts` directories.
2)  Building dependencies as usual with `carthage update` /  `carthage build`.
3)  Running `carthage build --no-skip-current --platform ios` from the `Carthage/Checkouts/facebook-objc-sdk` directory.

**Comment by macedonapp** [comment]
carthage update --platform iOS --no-use-binaries works for me

**Comment by RoryKelly** [comment]
`carthage update --platform iOS --no-use-binaries` works for me as well I think the issue is that the prebuilt binaries from facebook are not imported from the objective C SDK. `--no-use-binaries` ignores them and rebuilds them

**Comment by cgaisford** [comment]
`carthage update --platform iOS --no-use-binaries` also worked for me... finally!

**Comment by titopalito** [comment]
Thanks @macedonapp. it worked for me too.

**Comment by joesus** [comment]
@ValCanBuild is this still an issue we can help resolve? It seems like there is a clear way to use dynamic libraries with Carthage.

**Comment by dreamolight** [comment]
Updating dependency should fix the issue, we had broken carthage release on 5.3.0 of facebook-objc-sdk, and fixed since 5.4.0. As long as swift sdk is pulling version later or equal to 5.4.0, it should work.
